modules/ROOT/pages/cassandra/cassandra-design-center.adoc

```diff
@@ -0,0 +1,118 @@
+= Design Center - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+== Configure the Cassandra Connector in Design Center
+. From Anypoint Platform > Design Center, create a Mule Application, and add an HTTP Listener as the trigger.
++
+. Create an HTTP global element for the connector, by setting these fields:
++
+[%header%autowidth.spread]
+|===
+|Field |Description
+|Protocol | Protocol selected for the HTTP endpoint, it can be HTTP or HTTPS (secure).
+|Host| IP address where the Mule application listens for requests.
+|Port| Port address where the Mule application listens for requests.
+|Base Path| Path where the Mule application listens for requests.
+|===
++
+. Select the plus sign to add a component.
+. Select the connector as a component.
+. Select the operation.
++
+[%header%autowidth.spread]
+|===
+|Operation |Description
+|Add New Column a|Input for this operation is:
+
+* Parameter: A table name
+* Optional keyspace name
+* Payload: an AlterColumnInput POJO
++
+The POJO has the following fields:
++
+** Name of the column to be created
```

Comment:
Same comment about using or not using "the" at the start of the description.

Reply:
Noted, will fix in all.

Reply:
Fixed.